crates/bevy_pbr/src/material.rs

```diff
@@ -185,6 +205,10 @@ where
                 )
                 .add_system_to_stage(RenderStage::Queue, queue_material_meshes::<M>);
         }
+
+        if self.prepass_enabled {
+            app.add_plugin(PrepassPlugin::<M>::default());
```

Comment:
This is important and it's not obvious that this is being done by the MaterialPlugin. Should be called out in the docs!

Reply:
I'm not entirely sure which part you think should be mentioned. The part that this adds a plugin?

Reply:
Yep, exactly.